receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -483,9 +499,228 @@ metrics:
+    unit: 1
+    gauge:
+      value_type: int
+    attributes: [ ]
+    enabled: true
+  elasticsearch.node.cluster.published_states.full:
+    description: Number of published cluster states.
+    unit: 1
+    gauge:
+      value_type: int
+    attributes: [ ]
+    enabled: true
+  elasticsearch.node.cluster.published_states.incompatible:
+    description: Number of incompatible differences between published cluster states.
+    unit: 1
+    gauge:
+      value_type: int
+    attributes: [ ]
+    enabled: true
+  elasticsearch.node.cluster.published_states.compatible:
+    description: Number of compatible differences between published cluster states.
+    unit: 1
+    gauge:
+      value_type: int
+    attributes: [ ]
+    enabled: true
+  elasticsearch.node.cluster.state_update.count:
+    description: The number of cluster state update attempts that changed the cluster state since the node started.
+    unit: 1
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [ cluster_state_update_type ]
+    enabled: true
+  elasticsearch.node.cluster.state_update.computation_time:
+    description: The cumulative amount of time spent computing cluster state updates since the node started.
+    unit: 1
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [ cluster_state_update_type ]
+    enabled: true
+  elasticsearch.node.cluster.state_update.context_construction_time:
+    description: The cumulative amount of time spent constructing a publication context since the node started.
+    unit: 1
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [ cluster_state_update_type ]
+    enabled: true
+  elasticsearch.node.cluster.state_update.commit_time:
+    description: The cumulative amount of time spent waiting for a cluster state update to commit.
+    unit: 1
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [ cluster_state_update_type ]
+    enabled: true
+  elasticsearch.node.cluster.state_update.completion_time:
+    description: The cumulative amount of time spent waiting for a cluster state update to complete.
+    unit: 1
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [ cluster_state_update_type ]
+    enabled: true
+  elasticsearch.node.cluster.state_update.master_apply_time:
+    description: The cumulative amount of time spent applying cluster state updates on the elected master since the node started.
+    unit: 1
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [ cluster_state_update_type ]
+    enabled: true
+  elasticsearch.node.cluster.state_update.notification_time:
+    description: The cumulative amount of time spent notifying listeners of a cluster state update since the node started.
```

Comment:
Should these be combined? (If the sum of these times is a meaningful total, then most likely yes.)

Reply:
The sum of these metrics would likely just be the total time it took the cluster to take the action of updating something. I don't see this total time as meaningful?

Reply:
bump

Reply:
Missed this one. I'll combine these metrics together.